--- hamlib-3.3/tests/rotctl_parse.c.orig	2018-08-20 17:30:25.000000000 +0200
+++ hamlib-3.3/tests/rotctl_parse.c	2019-05-08 16:03:25.046301429 +0200
@@ -1009,7 +1009,7 @@
             /* The starting position of the source string is the first
              * character past the initial '\'.
              */
-            snprintf(cmd_name, sizeof(cmd_name), parsed_input[0] + 1);
+            snprintf(cmd_name, sizeof(cmd_name), "%s", parsed_input[0] + 1);
 
             /* Sanity check as valid multiple character commands consist of
              * alpha-numeric characters and the underscore ('_') character.
